srcpkgs/pgweb/template

```diff
@@ -9,7 +9,7 @@ maintainer="Gerardo Di Iorio <arete74@gmail.com>"
 license="MIT"
 homepage="https://sosedoff.github.io/pgweb"
```

Comment:
```suggestion
homepage="https://sosedoff.github.io/pgweb"
changelog="https://github.com/sosedoff/pgweb/raw/master/CHANGELOG.md"
```

Reply:
Thanks for suggest, i have update the pull request.

Reply:
I don't see it. did you push it?

Reply:
ipdate